devops/girder/annotation_client/annotation_client/utils.py

```diff
@@ -0,0 +1,17 @@
+import sys
+import json
+
+def sendProgress(progress, title, info):
```

Comment:
Is it really necessary as it only prints progress info and the method does not seem to be called ?

Reply:
The `stdout` of the worker is sent to the front end, this is why printing works 👍 
These method in the `annotation_client` are not called in this repo but they are used in the Workers themselves
For example, see [this commit](https://github.com/arjunrajlaboratory/ImageAnalysisProject/commit/beb094c8306cf71f1423ae4771afeafcdf08f977) which shows how to use it

Reply:
sounds good !